idl/github.com/uber/cadence/shared.thrift

```diff
@@ -368,6 +368,7 @@ struct WorkflowExecutionInfo {
   100: optional Memo memo
   101: optional SearchAttributes searchAttributes
   110: optional ResetPoints autoResetPoints
+  120: optional bool IsWorkflowOpen
```

Comment:
There are many other places that creating this WorkflowExecutionInfo struct. Make sure you change them all so that it won't be an issue when someone else needs to rely on this field.